SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -898,7 +898,31 @@ \subsubsection{Variable Initialization}
 following declaration would create a matrix with all zeros by default:
 \small
 \begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
+// Define the dimension of the images
+const unsigned int ImageDimension = 2;
 
+...
+
+// Declare the type of the index to access images
+typedef itk::Index< ImageDimension > IndexType;
```

Comment:
It is a best practice to declare the typedef next to where it is used. By keeping related logical sections closer, it prevents the need to search throughout the code for related sections and lowers cognitive overhead.